openpype/hosts/max/plugins/publish/extract_camera_fbx.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pyblish.api
+from openpype.pipeline import (
+    publish,
+    OptionalPyblishPluginMixin
+)
+from pymxs import runtime as rt
+from openpype.hosts.max.api import (
+    maintained_selection,
+    get_all_children
+)
+
+
+class ExtractCameraFbx(publish.Extractor,
```

Comment:
Is it weird maybe that this is `ExtractCameraFbx` but the other is `ExtractAlembicCamera`? :)

Should they either both be:
```
ExtractCameraAlembic
ExtractCameraFbx
```
or:
```
ExtractFbxCamera
ExtractAlembicCamera
```